src/app/containers/PageHandlers/withVariant/getVariantRedirectUrl/index.js

```diff
@@ -0,0 +1,51 @@
+import path from 'ramda/src/path';
+import pathToRegexp from 'path-to-regexp';
+import {
+  getVariant,
+  variantSanitiser,
+  variants,
+} from '#lib/utilities/variantHandler';
+import { getPreferredVariant } from '#app/contexts/UserContext/cookies';
+
+const getVariantRedirectUrl = (props, service, variant) => {
```

Comment:
you should change this signature to accept only what it needs rather than passing in all props

Reply:
Fixed!